CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Update from drush 11-12 to 13 and update command
```

Comment:
I suppose this is what we initially intended to do, but this should probably say something along the line of added drush 13 compatibility